cypress/e2e/marc-authority/marc-authority-edit.cy.js

```diff
@@ -11,6 +11,7 @@ import Logs from '../../support/fragments/data_import/logs/logs';
 import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
 import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
 import MarcFieldProtection from '../../support/fragments/settings/dataImport/marcFieldProtection';
+import { replaceByIndex } from '../../support/utils/stringTools';
```

Comment:
replaceByIndex working withot {}?

Reply:
No it doesn't work without it, because it is not in default export.